src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateIsLiked}) => {
+  const entryClass = (id % 2 === 0) ? 'remote':'local';
```

Comment:
Nice usage of the remainder operator here! This approach totally works, but it does assume that users alternate sending messages. Consider scenarios where users send multiple messages at a time. How could you refactor your code to handle this situation?